File: src/main/java/br/com/rotaractsorocabaleste/rotasales/saleitem/SaleItemService.java
package br.com.rotaractsorocabaleste.rotasales.saleitem;

import java.util.Set;
import java.util.UUID;

public interface SaleItemService {

    Set<SaleItem> saveAll(final Set<SaleItem> items);

    void deleteAllByIdInBatch(final Set<UUID> ids);

}

File: src/main/kotlin/br/com/rotaractsorocabaleste/rotasales/core/service/SaleItemService.kt
package br.com.rotaractsorocabaleste.rotasales.core.service

import br.com.rotaractsorocabaleste.rotasales.core.entity.SaleItem
import br.com.rotaractsorocabaleste.rotasales.core.vo.SaleRequestVO
import java.util.*

interface SaleItemService {
    fun saveSaleItems(saleItems: List<SaleItem>): List<SaleItem>
    fun getSaleItemsBySaleId(saleId: UUID): List<SaleItem>
    fun removeSaleItems(saleItems: List<SaleItem>)
}